rotkehlchen/tests/api/test_makerdao_dsr.py

```diff
@@ -117,10 +116,7 @@ def mock_etherscan_for_dsr(
 
     proxy1 = make_evm_address()
     proxy2 = make_evm_address()
-    targets = [address_to_32byteshexstr(proxy1), address_to_32byteshexstr(proxy2), '0x0000000000000000000000000000000000000000000000000000000000000000']  # noqa: E501
-
-    sorted_accounts = sorted(zip([account1, account2, account3], targets, strict=True), key=lambda x: x[0])  # noqa: E501
-    proxies = [y[1] for y in sorted_accounts]
+    proxies = [address_to_32byteshexstr(proxy1), address_to_32byteshexstr(proxy2), '0x0000000000000000000000000000000000000000000000000000000000000000']  # noqa: E501
```

Comment:
Why are you making changes to this test in this PR? Why is this affected by your changes?

Reply:
this has to change because since we changed the schema of blockchain_accounts table, now its SELECT query return the output in a different order for some reason.

```diff
diff --git a/rotkehlchen/db/dbhandler.py b/rotkehlchen/db/dbhandler.py
index da4555dde..a21327dfc 100644
--- a/rotkehlchen/db/dbhandler.py
+++ b/rotkehlchen/db/dbhandler.py
@@ -1262,7 +1262,7 @@ class DBHandler:
     def get_blockchain_accounts(self, cursor: 'DBCursor') -> BlockchainAccounts:
         """Returns a Blockchain accounts instance containing all blockchain account addresses"""
         cursor.execute(
-            'SELECT blockchain, account FROM blockchain_accounts;',
+            'SELECT blockchain, account FROM blockchain_accounts ORDER BY account;',
         )
         accounts_lists = defaultdict(list)
         for entry in cursor:
```
if we add this ^ change, then also the test is fixed. But that has a performance cost, so changing the order of the test
Order is not guaranteed in the absence of order by.